services/slam/fake/slam_test.go

```diff
@@ -311,38 +150,28 @@ func verifyGetPointCloudMapStreamStateful(t *testing.T, slamSvc *SLAM, extra map
 	getPositionResultsFirst := getPositionResults[len(getPositionResults)/2:]
 	getPositionResultsLast := getPositionResults[:len(getPositionResults)/2]
 
-	positionResultsFirst := positionResults[len(positionResults)/2:]
-	positionResultsLast := positionResults[:len(positionResults)/2]
-
-	getInternalStateResultsFirst := getInternalStateResults[len(getInternalStateResults)/2:]
-	getInternalStateResultsLast := getInternalStateResults[:len(getInternalStateResults)/2]
-
 	getInternalStateStreamResultsFirst := getInternalStateStreamResults[len(getInternalStateStreamResults)/2:]
 	getInternalStateStreamResultsLast := getInternalStateStreamResults[:len(getInternalStateStreamResults)/2]
 
+	getMapResultsFirst := getPointCloudMapResults[len(getPointCloudMapResults)/2:]
+	getMapResultsLast := getPointCloudMapResults[:len(getPointCloudMapResults)/2]
```

Comment:
[opt/nit] getPointCloudMapFirst/Last, but totally up to you

Reply:
done